datarepo-clienttests/src/main/resources/suites/NightlyPerfWorkflow.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "NightlyPerfWorkflow",
+  "description": "Performance tests to run on nightly github action",
+  "serverSpecificationFile": "perf.json",
+  "testConfigurationFiles": ["basicexamples/BasicAuthenticated.json", "basicexamples/BasicUnauthenticated.json", "basicexamples/LookupDataset.json", "profiling/BuildSnapshotWithFiles1000_50.json", "profiling/BulkLoadScale200_20.json"]
```

Comment:
@marikomedlock  Are there specific tests you would want me to add to this suite?

Reply:
Great question. I think we should discuss as a group. My opinion is that we should have at least 1 test here for each endpoint that we care about (e.g. DRS lookups, bulk file ingest, snapshot creation).